prow/prowjobs/GoogleCloudPlatform/gcp-guest/compute-image-tools.yaml

```diff
@@ -41,6 +41,24 @@ presubmits:
         command:
         - "/go/main.sh"
         args: ["cli_tools/"]
+  - name: cli-tools-presubmit-module-tests
+    cluster: gcp-guest
+    skip_report: true
```

Comment:
you won't see the result with this option. i think there is a better testing strategy than committing then separately enabling, and you should just submit the final enabled content, here. sync with me offline on the testing strategy